package.json

```diff
@@ -125,10 +125,12 @@
     "start:prod": "REACT_APP_ENV=prod node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
-    "lint:fix": "eslint --fix --",
+    "lint:fix": "eslint . --fix",
     "extract-messages": "NODE_ENV=production babel ./src  --out-file /dev/null",
     "manage:translations": "NODE_ENV=production ./node_modules/.bin/babel-node scripts/translator.js",
-    "build:langs": "npm run extract-messages && npm run manage:translations"
+    "build:langs": "npm run extract-messages && npm run manage:translations",
+    "precommmit": "npm run lint:fix",
+    "prepush": "npm run lint:fix"
```

Comment:
why not just have travis run `lint:fix`?

Reply:
explanation as above